script/bootstrap

```diff
@@ -17,6 +17,9 @@ process.on('unhandledRejection', function (e) {
   process.exit(1)
 })
 
+// We can't use yargs until installScriptDependencies() is executed, so...
+const ci = process.argv.indexOf('--ci') !== -1
```

Comment:
One alternative here is to look at `process.env.CI` and see if it's set to `true`, that'd save you from dealing with the command line parameter (and subsequenct .yml changes).  It's set by default on [AppVeyor](https://www.appveyor.com/docs/environment-variables/) and [Travis](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) and I've added it to the builds on VSTS too.

Reply:
Done 👍  I have it logging a message and allowing an override with `--no-ci` just in case 👌